src/vivarium_nih_us_cvd/results_processing/process_results.py

```diff
@@ -57,17 +57,17 @@ class MeasureData(NamedTuple):
     deaths: pd.DataFrame
     state_person_time: pd.DataFrame
     transition_count: pd.DataFrame
-    risk_exposure_time: pd.DataFrame
+    # risk_exposure_time: pd.DataFrame
     binned_ldl_exposure_time: pd.DataFrame
     binned_sbp_exposure_time: pd.DataFrame
-    healthcare_visits: pd.DataFrame
-    sbp_medication_person_time: pd.DataFrame
-    ldlc_medication_person_time: pd.DataFrame
-    intervention_person_time: pd.DataFrame
+    # healthcare_visits: pd.DataFrame
+    # sbp_medication_person_time: pd.DataFrame
+    # ldlc_medication_person_time: pd.DataFrame
+    # intervention_person_time: pd.DataFrame
 
     def dump(self, output_dir: Path):
         for key, df in self._asdict().items():
-            df.to_hdf(output_dir / f"{key}.hdf", key=key)
+            # df.to_hdf(output_dir / f"{key}.hdf", key=key)
```

Comment:
Is this broken or are hdf files not desired (for now)?

Reply:
Both. transition_count.hdf is and has always been broken (and Hussain has confirmed he sees it as well in mnch). Syl also confirmed she never reads the hdfs.